.github/workflows/codeql-analysis.yml

```diff
@@ -7,8 +7,15 @@ on:
     - cron: '0 0 * * *'
     - cron: '0 12 * * *'
 
+permissions:
+  contents: read
+
 jobs:
   analyze:
+    permissions:
+      actions: read  # for github/codeql-action/init to get workflow details
+      contents: read  # for actions/checkout to fetch code
+      security-events: write  # for github/codeql-action/analyze to upload SARIF results
```

Comment:
CodeQL is currently manually disabled in the repo. Can you please remove this write?

Reply:
I have removed that, Thanks!